web/src/components/RemoteContentSandBox.ts

```diff
@@ -144,6 +144,38 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     font-weight: bold;
   }
 
+  .contact-card {
+    display: inline-block;
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: 0.25rem;
+    background-repeat: no-repeat;
+    background-color: rgb(127 127 127 / 15%) !important;
```

Comment:
:wrench: It does not seem like the `!important` is needed, therefore please remove. Same for the image below and for native.